pom.xml

```diff
@@ -105,30 +105,6 @@
 				<type>jar</type>
 				<scope>compile</scope>
 			</dependency>
-			<dependency>
-				<groupId>junit</groupId>
-				<artifactId>junit</artifactId>
-				<version>4.11</version>
-				<scope>test</scope>
-			</dependency>
-			<dependency>
-				<groupId>org.hamcrest</groupId>
-				<artifactId>hamcrest-core</artifactId>
-				<version>1.3</version>
```

Comment:
hamcrest here is on 1.3 and the version in openmrs-test is 2.2 I think.

Reply:
@teleivo this is the result . And from the result the is no junit 5 dependencies . I don 't know what can be the problem . 
![Screenshot from 2020-08-07 12-31-11](https://user-images.githubusercontent.com/39851246/89641659-1402f700-d8aa-11ea-8ac0-0738301c7f86.png)

Reply:
I have been try to fix it without succeed.

Reply:
we should rule out that its not an old version that is causing this. since with SNAPSHOT versions the content in the artifact van change. So maybe you have an older version of the openmrs-test locally before we merged the change in openmrs-core. You can try adding a `-U` to the dependency command, or any other command you are running to make sure the snapshot is updated. See https://stackoverflow.com/questions/29020716/maven-what-does-u-update-snapshots-really-do

Reply:
is it working for you now that the PR is merged and with my suggestion?

Reply:
> hamcrest here is on 1.3 and the version in openmrs-test is 2.2 I think.

yes , there is no need to add or upgrade hanccerst, since it  is provide in org.openmrs.test:openmrs-test:pom.